pkgs/development/libraries/folly/default.nix

```diff
@@ -23,13 +23,13 @@
 
 stdenv.mkDerivation rec {
   pname = "folly";
-  version = "2023.02.27.00";
+  version = "2023.10.02.00";
```

Comment:
2023.10.30 has been tagged by now.

Reply:
Also, 2024.01.01.00 is out.

Reply:
Upgraded to: 2024.01.01.00
/cc @wegank @FliegendeWurst